app/tests/integration/conftest.py

```diff
@@ -142,132 +185,100 @@ def fill_design_page(selenium, base_url, request, variables, fill_overview):
 
 @pytest.fixture
 def fill_design_page_multi_prefs(
-    selenium, base_url, request, variables, experiment_type, fill_overview
+    selenium, base_url, request, default_data, experiment_type, fill_overview
 ):
     """Fills design page according to multi pref requirements."""
     design = DesignPage(selenium, base_url, experiment_url=f"{fill_overview.url}").open()
     design = design.wait_for_page_to_load()
     design.enable_multipref()
     branches = design.current_branches
-    for count, branch in enumerate(variables[experiment_type]["branches"]):
-        branches[count].branch_name = f"{branch['branch_name']}"
-        branches[count].branch_description = f"{branch['branch_description']}"
+    for count, branch in enumerate(default_data.branches):
+        branches[count].branch_name = branch.branch_name
+        branches[count].branch_description = branch.branch_description
         branches[count].add_pref_button.click()
         prefs = branches[count].prefs(count)
-        for pref_num in range(0, len(branch["preferences"])):  # Fill in multi prefs
-            preferences = branch["preferences"][pref_num]
-            prefs[pref_num].pref_branch = preferences["firefox_pref_branch"]
-            prefs[pref_num].pref_type = preferences["firefox_pref_type"]
-            prefs[pref_num].pref_name = preferences["firefox_pref_name"]
-            prefs[pref_num].pref_value = preferences["firefox_pref_value"]
+        for pref_num in range(0, len(branch.preferences)):  # Fill in multi prefs
+            preferences = branch.preferences[pref_num]
```

Comment:
Couldn't this be `for preference in branch.preferences`?

Reply:
Tis could work with an ```enumerate()```